packages/graphql/src/translate/cypher-builder/Pattern.ts

```diff
@@ -157,4 +157,15 @@ export class Pattern<T extends MatchableElement = any> implements CypherCompilab
 
         return padLeft(stringifyObject(paramValues));
     }
+
+    private getNodeLabelsString(node: NodeRef): string {
+        const escapedLabels = node.labels.map(escapeLabel);
+        if (escapedLabels.length === 0) return "";
+        return `:${escapedLabels.join(":")}`;
+    }
+
+    private getRelationshipLabelsString(relationship: RelationshipRef): string {
+        // TODO: escapeLabel
```

Comment:
question (non-blocking): for my understanding, is the escaping of node labels any different from the escaping of rel types?

Reply:
Nope, but due to it being not a high priority and a breaking change we are not escaping them until next major release